core/src/stories/patterns/navigation/navigation-manyitems.stories.tsx

```diff
@@ -154,7 +154,8 @@ const Template = ({ dummyHtml }) =>
       </tds-header-dropdown>
       
       
-      <tds-header-brand-symbol slot="end" link-href="https://design.scania.com" aria-label="Scania - red gryphon on blue shield">
+      <tds-header-brand-symbol slot="end" aria-label="Scania - red gryphon on blue shield">
```

Comment:
The aria label should be moved to the `<a>` tag since that is the element with the image.